.github/workflows/label_and_milestone_checker.yml

```
@@ -82,7 +82,7 @@ jobs:
           PR_AFFILIATION: ${{ github.event.pull_request.user.company }}
         run: |
           echo "PR author: $PR_AUTHOR"
-          if [ "$PR_AUTHOR" = "napari-bot" ] || [ "$PR_AUTHOR" = "pre-commit-ci" ] || [ "$PR_AUTHOR" = "dependabot" ]; then
+          if [[ "$PR_AUTHOR" == *"[bot]" ]]; then
```

```suggestion
          if [[ "$PR_AUTHOR" == *"[bot]" || "$PR_AUTHOR" == *-bot ]]; then
```
I think we still want to catch `napari-bot` right?

---

napari-bot will appear as `napari-bot[bot]` in here, so it will be caught!

---

It didn't appear that way in the pr I checked...

---

My bad, you're right! https://github.com/napari/napari/actions/runs/19336684586/job/55313477627?pr=8310

Why si it different? Isn't it marked as a bot?

---

In that case I think it's best to explicitly check for `napari-bot` rather than `*-bot`.